### ISSUE TITLE
Check if banned members competed when they could not

### ISSUE THREAD

**Issue by pedrosino** [thewca/worldcubeassociation.org#1037]
We do have a list of banned people (and how long the ban lasts, of course), but we don't have a way to check this on the posting results process.

**Comment by jfly** [comment]
idea, but might work... @pedrosino, do you think we could use the teams infrastructure to maintain a list of banned members? Team members have start and end dates. That would be almost 0 work to implement, it would just require every banned member to have a WCA website account.

**Comment by pedrosino** [comment]
I'm just worried about the only requirement. Even though we have more than 35k users, only 13545 (~37.8%) have an assigned WCA id, and almost 6k (~16.5%) are not even confirmed accounts.
It would be very easy to implement, but probably not have the desired reach.

**Comment by jfly** [comment]
Yeah, we'd have to create accounts for them ourselves if they don't already have one. This is a bit of manual work, but I don't think it's a problem.

As more and more completions use the wca website for registration, this becomes less of an issue, as well.

**Comment by pedrosino** [comment]
You mean like the dummy accounts we have now for people with an avatar? I guess that could work.

**Comment by jfly** [comment]
Yeah, something like that.

**Comment by AlbertoPdRF** [comment]
Should this check also be added to https://github.com/thewca/worldcubeassociation.org/projects/4?

**Comment by jfly** [comment]
Yes, I think so! I've gone ahead and added it =)

**Comment by Jambrose777** [comment]
This could be handled either at results upload or during the WRT sanity Check. (Probably should be checked in both).
